Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -0,0 +1,301 @@
+.desktop {
+  background-color: #{$background-color};
+
+  @media #{$breakpoint-mobile} {
+    display: none;
+  }
+}
+
+.mobile {
+  background-color: #{$background-color};
+
+  @media #{$breakpoint-tablet} {
+    display: none;
+  }
+}
+
+.display-none {
+  display: none;
+}
+
+.info-dialog {
+  align-items: center;
+  background-color: #{$info-dialog-color};
+  display: flex;
+  height: 35px;
+  justify-content: center;
+  position: relative;
+
+  p {
+    color: #{$color-body};
+    font-family: 'Oswald', sans-serif;
+    font-size: 12px;
+    margin: 2px;
+  }
+
+  .dialog-button-container {
+    align-items: flex-start;
+    display: flex;
+    justify-content: flex-end;
+    position: absolute;
+    right: 5px;
+    top: 7px;
+    width: 100%;
+
+    button {
+      background-color: #{$color-body};
+      border-radius: 50%;
+      border-style: none;
+      color: #abc828;
+      font-weight: bold;
+      height: 20px;
+      text-align: center;
+      width: 20px;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+}
+
+.current-route-container {
+  background-color: #{$background-color};
+  padding-top: 10px;
```

Comment:
Properties should be ordered background-color, padding-bottom, padding-top, width